.travis.yml

```diff
@@ -1,31 +1,59 @@
 sudo: false
 language: scala
 
-scala:
-  - 2.12.8
-  - 2.11.12
+scala_version_211: &scala_version_211 2.11.12
+scala_version_212: &scala_version_212 2.12.8
+scala_version_213: &scala_version_213 2.13.0-M5
 
-jdk:
-  - openjdk11
-  - openjdk12
+java11: &java_11 openjdk11
+java12: &java_12 openjdk12
 
-before_install:
-  - export PATH=${PATH}:./vendor/bundle
+stages:
+  - name: test
+  - name: publish
+    if: (branch = master AND type = push)
 
-install:
-  - rvm use 2.6.0 --install --fuzzy
-  - gem update --system
-  - gem install sass
-  - gem install jekyll -v 3.2.1
+jobs:
+  include:
+    - &tests
+      env: TEST="test"
+      script:
+        - sbt ++$TRAVIS_SCALA_VERSION test
+        - sbt ++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues
+      scala: *scala_version_211
+      jdk: *java_11
+    - <<: *tests
+      scala: *scala_version_212
+      jdk: *java_11
+    - <<: *tests
+      scala: *scala_version_213
+      jdk: *java_11
+    - <<: *tests
+      scala: *scala_version_212
+      jdk: *java_12
 
-script:
-  - sbt ++$TRAVIS_SCALA_VERSION test
-  - sbt ++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues
-  - sbt ++$TRAVIS_SCALA_VERSION docs/makeMicrosite
+    - env: TEST="docs"
+      before_install:
+        - export PATH=${PATH}:./vendor/bundle
+      install:
+        - rvm use 2.6.0 --install --fuzzy
+        - gem update --system
+        - gem install sass
+        - gem install jekyll -v 3.2.1
+      script: sbt ++$TRAVIS_SCALA_VERSION docs/makeMicrosite
+      scala: *scala_version_212
 
-after_success:
-  - test $TRAVIS_PULL_REQUEST == "false" && test $TRAVIS_JDK_VERSION == "openjdk11" && test $TRAVIS_BRANCH == "master" && test $TRAVIS_REPO_SLUG == "http4s/http4s-jdk-http-client" && sbt ++$TRAVIS_SCALA_VERSION publish
-  - test $TRAVIS_PULL_REQUEST == "false" && test $TRAVIS_JDK_VERSION == "openjdk11" && test $TRAVIS_BRANCH == "master" && test $TRAVIS_REPO_SLUG == "http4s/http4s-jdk-http-client" && test $TRAVIS_SCALA_VERSION == "2.12.8" && sbt docs/publishMicrosite
+    - stage: publish
+      env: TEST="publish"
+      scala: *scala_version_212
```

Comment:
Don't I need one of these for every Scala version, too?  Or can I just say `+publish` here?  I'm getting dizzy.